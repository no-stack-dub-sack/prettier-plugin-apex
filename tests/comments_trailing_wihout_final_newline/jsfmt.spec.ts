runSpec(__dirname, ["apex"], {
  tabWidth: 2,
  apexInsertFinalNewline: false,
});
