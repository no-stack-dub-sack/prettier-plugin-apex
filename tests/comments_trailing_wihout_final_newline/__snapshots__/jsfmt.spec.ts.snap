// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex: ClassDeclarationWithBlockComment.cls: ClassDeclarationWithBlockComment.cls 1`] = `
class ClassDeclarationWithBlockComment {
}
/* Test comment */
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class ClassDeclarationWithBlockComment {
}
/* Test comment */
`;

exports[`Format apex: ClassDeclarationWithInlineComment.cls: ClassDeclarationWithInlineComment.cls 1`] = `
class ClassDeclarationWithInlineComment {
}
// Test comment
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class ClassDeclarationWithInlineComment {
}
// Test comment
`;

exports[`Format apex: ClassDeclarationWithMixedComments.cls: ClassDeclarationWithMixedComments.cls 1`] = `
class ClassDeclarationWithMixedComments {
} // Trailing Comment
/* Block comment 1 */
// Another Trailing Comment
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class ClassDeclarationWithMixedComments {
} // Trailing Comment
/* Block comment 1 */
// Another Trailing Comment
`;
